### PR TITLE
Update README & Add Matchit plugin support 

### PR DESCRIPTION
**Pull request by jyyan**
Additional support Matchit.vim plugin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# vim-volt-syntax
+Simple **Volt** syntax based on **Jinja2** highlighting
+
+Additional support [**Matchit.vim**](https://github.com/vim-scripts/matchit.zip) plugin
```
Comment (etaoins) [inline README.md]: Maybe change this to:

```
Additionally supports the [**Matchit.vim**](https://github.com/vim-scripts/matchit.zip) plugin
```

**Comment by etaoins** [comment]
Looks really good! Happy to merge after the grammar fix. Thanks for doing this.

File: README.md (hunk 1 above)
Comment (jyyan) [inline README.md]: okay , thanks for your help , will update it soon :P

**Comment by etaoins** [comment]
:rocket:

**Comment by jyyan** [comment]
thanks for your help  :P

**Pull request merged**